packages/fast-components-react-base/src/viewport-positioner/README.md

```diff
@@ -40,6 +42,10 @@ The "AlwaysInView" props (i.e. "horizontalAlwaysInView" and "verticalAlwaysInVie
 
 The "fixedAfterInitialPlacement" prop is set the component will not adjust positioning after the initial render.
 
+Authors should be mindful that instanciating a flyout positioner can cause layout changes as it is actually inserted into the dom and can cause parent containers to expand, move siblings down the page, etc...  It is up to authors to place the component such that it does not cause unwanted layout changes.  
```

Comment:
nit: "dom" should be all caps as it is an acronym.

Reply:
yep